cmd/minishift/cmd/oc_env.go

```diff
@@ -132,8 +131,7 @@ func init() {
 // Because InstanceConfig.OcPath is set in minishift start or profile set. So when oc-env is called with --profile
 // it points to last minishift start or profile set.
 func getOcPath() string {
-	requestedOpenShiftVersion := viper.GetString(configCmd.OpenshiftVersion.Name)
-	ocVersion := clusterup.DetermineOcVersion(requestedOpenShiftVersion)
+	ocVersion := viper.GetString(configCmd.OpenshiftVersion.Name)
```

Comment:
@praveenkumar How did this work? `configCmd.OpenshiftVersion` is not a flag in `oc-env` command. Even if Viper magic works here, it is worng, isn't it?

Reply:
@LalatenduMohanty what's wrong here? it is the same way other configCmd works we just moved OpenshiftVersion flag from the clusterUp to start flags. What is the confusion?

Reply:
@praveenkumar we are getting a value from a flag `--openshift-version` here right?

Reply:
@LalatenduMohanty That's right

Reply:
Added https://github.com/minishift/minishift/issues/2389#issuecomment-389871088